docs/developer-docs/integrations/sns/integrate-sns/index-integration.md

```diff
@@ -1 +1,51 @@
-Title h1: Considerations to Integrate with Index Canister
\ No newline at end of file
+# Index Canister
+
+The index canister keeps track of each transaction stored by the 
+[ledger canister](ledger-integration.md).
+
+At each heartbeat, the index canister will query the transactions from
+the ledger canister, then build the index
+of known transaction per account.
+
+## Initialisation
+
+The index canister initialisation requires the principal
+of the ledger canister to index:
```

Comment:
Does any ledger work?
Does it have to be an ICRC-1 ledger canister?
Should we maybe explain this here?